dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/TrackerExportTests.java

```diff
@@ -284,4 +292,52 @@ public void shouldReturnAscOrderedEventByTEIAttribute()
         List<String> events = response.extractList( "instances.event.flatten()" );
         assertEquals( List.of( "ZwwuwNp6gVd", "olfXZzSGacW" ), events, "Events are not in the correct order" );
     }
+
+    @Test
+    void getTeiByPotentialDuplicateParamNull()
```

Comment:
We agreed to use the test name convention: _shouldXXWhenYYY_, is there a reason not to use something like `shouldReturnPotentialDuplicatePropertyWhenParamNull` in this method? 

The same applies for the ones below.

Reply:
yes, I know but that is before we agreed on the naming convention and in master, it is already merged that way. So I'll leave it like this for now